content/numpy/concepts/built-in-functions/terms/std/std.md

```diff
@@ -0,0 +1,102 @@
+---
+Title: '.std()'
+Description: 'Calculates the standard deviation of given data along a specified axis'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Arrays'
+  - 'Functions'
+  - 'NumPy'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+The **`.std()`** function returns the standard deviation of the array elements. The standard deviation is calculated for the flattened array by default, otherwise over the specified axis.
```

Comment:
You need to reword this part.

It shows up as directly plagiarized from https://www.javatpoint.com/numpy-std and https://numpy.org/devdocs/reference/generated/numpy.nanstd.html